src/tool/cpp/strings/base_com_ptr.h

```diff
@@ -8,6 +8,10 @@ WINRT_EXPORT namespace winrt
 
         com_ptr(std::nullptr_t = nullptr) noexcept {}
 
+        com_ptr(take_ownership_from_abi_t, void* ptr) noexcept : m_ptr(static_cast<type*>(ptr))
```

Comment:
>take_ownership_from_abi_t [](start = 16, length = 25)

this could come in handy with the new error object.  ;-)

Reply:
yep 😉